examples/convert_ieeg_to_bids.py

```diff
@@ -0,0 +1,170 @@
+"""
+===================================
+07. Convert iEEG data to BIDS format
+===================================
+
+In this example, we use MNE-BIDS to create a BIDS-compatible directory of EEG
+data. Specifically, we will follow these steps:
+
+1. Download some iEEG data from the MNE-Python API
+   `PhysioBank database <https://physionet.org/physiobank/database>`_.
+    `https://mne.tools/stable/auto_tutorials/misc/plot_ecog.html#sphx-glr-auto-tutorials-misc-plot-ecog-py`_.
+
+2. Load the data, extract information, and save in a new BIDS directory
+
+3. Check the result and compare it with the standard
+
+The iEEG data will be pretty similar to the EEG data with the addition of extra elements in
+the electrodes.tsv and coord_system.json files.
+"""
+
+# Authors: Adam Li <adam2392@gmail.com>
+# License: BSD (3-clause)
+
+import os
+import tempfile
+
+import mne
+import numpy as np
+from scipy.io import loadmat
+
+from mne_bids import write_raw_bids, make_bids_basename, read_raw_bids
+from mne_bids.utils import print_dir_tree
+
+# need to install visualization
+from mpl_toolkits import mplot3d
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Step 1: Download the data
+# -------------------------
+#
+# First, we need some data to work with. We will use the
+# data downloaded via MNE-Python's API.
+#
+# `https://mne.tools/stable/generated/mne.datasets.misc.data_path.html#mne.datasets.misc.data_path`_.
+#
+# Conveniently, there is already a data loading function available with
+# MNE-Python:
+
+# Make a directory to save the data to
+home = os.path.expanduser('~')
+mne_dir = os.path.join(home, 'mne_data')
+if not os.path.exists(mne_dir):
+    os.makedirs(mne_dir)
+
+mat = loadmat(mne.datasets.misc.data_path() + '/ecog/sample_ecog.mat')
+ch_names = mat['ch_names'].tolist()
+elec = mat['elec']  # electrode positions given in meters
+# Now we make a montage stating that the sEEG contacts are in head
+# coordinate system (although they are in MRI). This is compensated
+# by the fact that below we do not specicty a trans file so the Head<->MRI
+# transform is the identity.
+montage = mne.channels.make_dig_montage(ch_pos=dict(zip(ch_names, elec)),
+                                        coord_frame='head')
```

Comment:
can we avoid this hack here? I suspect you should save the iEEG channel locations in the MRI coordinate
system.

Reply:
How can we do this?

I'm following this tutorial in MNE-Python: https://mne.tools/dev/auto_tutorials/misc/plot_ecog.html#sphx-glr-auto-tutorials-misc-plot-ecog-py

Reply:
yes but this tutorial uses a hack as the comment says. Also see that you refer to a trans that is not actually there in this example. @sappelhoff anything to suggest to be able to write electrode locations in MRI space in a bids dataset? I fear that writing electrode locations in neuromag/mne head coordinate system is going into a too MNE centered direction.

Reply:
This might be more easily answered by @choldgraf --> Chris, how do we save iEEG electrode coordinates in MRI space in a BIDS dataset?